strawberry/directive.py

```diff
@@ -19,6 +19,12 @@
 from strawberry.unset import UNSET
 
 
+if sys.version_info < (3, 8):
+    from backports.cached_property import cached_property
+else:
+    from functools import cached_property
```

Comment:
could we make our own helper that does this? so we don't have to repeat in multiple places?

Reply:
Good idea, where would you suggest putting that?

Reply:
`utils/cached_property.py` sounds like a good enough place 😊